modules/util/source-formatter/src/main/resources/dependencies/imports.txt

```diff
@@ -2161,6 +2162,7 @@ com.liferay.portlet.wiki.service.persistence.WikiPageUtil=com.liferay.wiki.servi
 com.liferay.portlet.wiki.util.comparator.PageCreateDateComparator=com.liferay.wiki.util.comparator.PageCreateDateComparator
 com.liferay.portlet.wiki.util.comparator.PageTitleComparator=com.liferay.wiki.util.comparator.PageTitleComparator
 com.liferay.portlet.wiki.util.comparator.PageVersionComparator=com.liferay.wiki.util.comparator.PageVersionComparator
+com.liferay.product.navigation.personal.menu.configuration.PersonalMenuConfigurationTracker=com.liferay.product.navigation.personal.menu.configuration.PersonalMenuConfigurationRegistry
```

Comment:
@TamyrisTorres to add imports on imports.txt is necessary to add in alphabetic order (case sensitive)